crawler/url_crawler.go

```
@@ -1,182 +1,470 @@
-	c.OnRequest(func(r *colly.Request) {
-		log.Info().Msg("Visiting: " + r.URL.String())
-	})
-	c.Visit(url)
-	return lastPage
+	return nil
+}
 
+func (c *CrawlerImpl) Crawl(ctx context.Context, url string) error {
+	page := c.browser.MustPage(url)
+
+	c.crawlJudgement(ctx, page, url)
```

golangci-lint (1.62.2)

Error return value of `c.crawlJudgement` is not checked

(errcheck)